<!DOCTYPE HTML>
<!--
	Dimension by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Dr George Busby</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="logo">
							<span class="icon fa-gem"></span>
						</div>
						<div class="content">
							<div class="inner">
								<h1>Dr George Busby</h1>
								<p>geneticist and innovation executive</p>
							</div>
						</div>
						<nav>
							<ul>
								<li><a href="#intro">About</a></li>
								<li><a href="#about">CV</a></li>
								<li><a href="#projects">Projects</a></li>
								<li><a href="#papers">Publications</a></li>
								<!--<li><a href="#contact">Contact</a></li>-->
								<!--<li><a href="#elements">Elements</a></li>-->
							</ul>
						</nav>
					</header>

				<!-- Main -->
					<div id="main">

						<!-- Intro -->
							<article id="intro">
								<h2 class="major">About</h2>
								<span class="image main"><img src="images/pic01.jpg" alt="" /></span>
								<p>As a geneticist and leader in translational genomics, my career has been dedicated to bridging the gap between cutting-edge genetic research and real-world clinical applications. With a focus on precision medicine, I have successfully led the development and integration of genetic risk prediction tools into healthcare systems globally, overseeing the design and execution of commercial products that provide actionable insights for clinicians. A key part of this work has been working with end-users to iterate and develop our products so that they serve the required purpose, whilst maintaining sufficient scientific rigour to be clinically viable. We have put our products into people’s hands and pioneered the clinical use of common genetic variation for public health. We have shown that whilst rare variants are still important for understanding disease risk, they are not the only players in town. Future use of genomics across healthcare will need to incorporate both rare and common variation.
								</p>
								<p>I am currently CSO at <a href="https://allelica.com">Allelica</a> where I lead our R&D efforts. Allelica builds software to enable clinical genomics labs and healthcare systems to run clinical polygenic risk scores.</p> You can find a record of my academic publications <a href="#papers">here </a>.</p>

							</article>

						<!-- Projects -->
						<article id="projects">
							<h2 class="major">Projects</h2>
							<span class="image main"><img src="images/pic04.jpg" alt="" /></span>
							<p>Here is a selection of some of the projects I have worked on over the years.</p>

							<div class="table-wrapper">
								<table>
									
									<tbody>
										<tr>
											<td>NOMADS</td>
											<td>Optimising a nanopore‐based assay for
												malaria resistance markers</td>
											<td>May 2020 – Feb 2022</td>
										</tr>
										<tr>
											<td>The 2018 RGS Land Rover Bursary </td>
											<td>The Mobile Malaria Project: £30,000 and loan of a Land Rover Discovery for The Mobile Malaria Project. This was a unique expedition to understand the status of malaria research in Africa and to trial mobile genetic sequencing technology in low resource settings. <a href="https://mobilemalaria.com/resources/comic/">mobilemalaria.com</a></td>
											<td>Jan 2018 - Sep 2019</td>
										</tr>
										<tr>
											<td>The Mobile Malaria Project Expedition Comic Strip</td>
											<td>£4,850 Public Engagement with Research Grant from Oxford University to develop a comic strip about malaria and Africa, with Oxford‐based children’s comic, The Phoenix. You can see it <a href="https://mobilemalaria.com/resources/comic/">here.</a></td>
											<td>Jul 2017‐Jul 2018</td>
										<tr>
											<td>Genomes the play</td>
											<td>£15,000 Wellcome/Oxford University PE Grant to create a play about big data and medicine. I planned and executed an innovative public engagement event involving a play about the future of healthcare. You can see the report <a href="https://drive.google.com/open?id=1CboavCFmJTKtrJ3vK6ejB0aQO9FXllp9">here.</a></td>
											<td>Jul 2017‐Jul 2018</td>
										</tr>
									</tbody>
								</table>
							</div>



						</article>	

						<!-- Research -->
						
							<article id="papers">
								<h2 class="major">Publications</h2>
								<span class="image main"><img src="images/pic02.jpg" alt="" /></span>

								<p><div class="csl-bib-body">
									<div data-csl-entry-id="busby_genetic_2009" class="csl-entry">
									  <div class="csl-left-margin">[1]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b> <i>et al.</i>, “Genetic analysis of scat reveals leopard Panthera pardus and cheetah Acinonyx jubatus in southern Algeria,” <i><b>Oryx</b></i>, vol. 43, no. 3, pp. 412–415, 2009.</div>
									</div>
									<div data-csl-entry-id="busby_peopling_2012" class="csl-entry">
									  <div class="csl-left-margin">[2]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b> <i>et al.</i>, “The peopling of Europe and the cautionary tale of Y chromosome lineage R-M269,” <i><b>Proceedings of the Royal Society B: Biological Sciences</b></i>, vol. 279, no. 1730, pp. 884–892, Mar. 2012, doi: 10.1098/rspb.2011.1044.</div>
									</div>
									<div data-csl-entry-id="busby_finding_2006" class="csl-entry">
									  <div class="csl-left-margin">[3]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b>, “Finding the Blues: an investigation into the origin and evolution of African American music,” MRes, Imperial College, London,UK, 2006.</div>
									</div>
									<div data-csl-entry-id="bertoncini_y_2012" class="csl-entry">
									  <div class="csl-left-margin">[4]</div><div class="csl-right-inline">S. Bertoncini <i>et al.</i>, “A Y variant which traces the genetic heritage of ligures tribes,” <i><b>Journal of Biological Research - Bollettino della Società Italiana di Biologia Sperimentale</b></i>, vol. 85, no. 1, Nov. 2012, doi: 10.4081/jbr.2012.4087.</div>
									</div>
									<div data-csl-entry-id="coia_demographic_2013" class="csl-entry">
									  <div class="csl-left-margin">[5]</div><div class="csl-right-inline">V. Coia <i>et al.</i>, “Demographic Histories, Isolation and Social Factors as Determinants of the Genetic Structure of Alpine Linguistic Groups,” <i><b>PLoS ONE</b></i>, vol. 8, no. 12, p. e81704, Dec. 2013, doi: 10.1371/journal.pone.0081704.</div>
									</div>
									<div data-csl-entry-id="hellenthal_genetic_2014" class="csl-entry">
									  <div class="csl-left-margin">[6]</div><div class="csl-right-inline">G. Hellenthal <i>et al.</i>, “A Genetic Atlas of Human Admixture History,” <i><b>Science</b></i>, vol. 343, no. 6172, pp. 747–751, Feb. 2014, doi: 10.1126/science.1243518.</div>
									</div>
									<div data-csl-entry-id="marks_static_2014" class="csl-entry">
									  <div class="csl-left-margin">[7]</div><div class="csl-right-inline">S. J. Marks <i>et al.</i>, “Static and moving frontiers: the genetic landscape of Southern African Bantu-speaking populations,” <i><b>Molecular Biology and Evolution</b></i>, p. msu263, Sep. 2014, doi: 10.1093/molbev/msu263.</div>
									</div>
									<div data-csl-entry-id="lazaridis_ancient_2014" class="csl-entry">
									  <div class="csl-left-margin">[8]</div><div class="csl-right-inline">I. Lazaridis <i>et al.</i>, “Ancient human genomes suggest three ancestral populations for present-day Europeans,” <i><b>Nature</b></i>, vol. 513, no. 7518, pp. 409–413, Sep. 2014, doi: 10.1038/nature13673.</div>
									</div>
									<div data-csl-entry-id="cini_social_2015" class="csl-entry">
									  <div class="csl-left-margin">[9]</div><div class="csl-right-inline">A. Cini, S. Patalano, A. Segonds-Pichon, <b>G.</b> <b>Busby</b>, R. Cervo, and S. Sumner, “Social parasitism and the molecular basis of phenotypic evolution,” <i><b>Frontiers in Genetics</b></i>, vol. 6, p. 32, 2015, doi: 10.3389/fgene.2015.00032.</div>
									</div>
									<div data-csl-entry-id="montinaro_unravelling_2015" class="csl-entry">
									  <div class="csl-left-margin">[10]</div><div class="csl-right-inline">F. Montinaro, <b>G.</b> <b>Busby</b>, V. L. Pascali, S. Myers, G. Hellenthal, and C. Capelli, “Unravelling the hidden ancestry of American admixed populations,” <i><b>Nature Communications</b></i>, vol. 6, Mar. 2015, doi: 10.1038/ncomms7596.</div>
									</div>
									<div data-csl-entry-id="tofanelli_greeks_2015" class="csl-entry">
									  <div class="csl-left-margin">[11]</div><div class="csl-right-inline">S. Tofanelli <i>et al.</i>, “The Greeks in the West: genetic signatures of the Hellenic colonisation in southern Italy and Sicily,” <i><b>European Journal of Human Genetics</b></i>, Jul. 2015, doi: 10.1038/ejhg.2015.124.</div>
									</div>
									<div data-csl-entry-id="martinez-cadenas_relationship_2015" class="csl-entry">
									  <div class="csl-left-margin">[12]</div><div class="csl-right-inline">C. Martinez-Cadenas <i>et al.</i>, “The relationship between surname frequency and Y chromosome variation in Spain,” <i><b>European Journal of Human Genetics</b></i>, Apr. 2015, doi: 10.1038/ejhg.2015.75.</div>
									</div>
									<div data-csl-entry-id="busby_role_2015" class="csl-entry">
									  <div class="csl-left-margin">[13]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b> <i>et al.</i>, “The Role of Recent Admixture in Forming the Contemporary West Eurasian Genomic Landscape,” <i><b>Current Biology</b></i>, vol. 25, no. 19, pp. 2518–2526, May 2015, doi: 10.1016/j.cub.2015.08.007.</div>
									</div>
									<div data-csl-entry-id="gonzalez-santos_genome-wide_2015" class="csl-entry">
									  <div class="csl-left-margin">[14]</div><div class="csl-right-inline">M. González-Santos <i>et al.</i>, “Genome-Wide SNP Analysis of Southern African Populations Provides New Insights into the Dispersal of Bantu-Speaking Groups,” <i><b>Genome Biology and Evolution</b></i>, vol. 7, no. 9, pp. 2560–2568, Sep. 2015, doi: 10.1093/gbe/evv164.</div>
									</div>
									<div data-csl-entry-id="hellenthal_kalash_2016" class="csl-entry">
									  <div class="csl-left-margin">[15]</div><div class="csl-right-inline">G. Hellenthal <i>et al.</i>, “The Kalash Genetic Isolate? The Evidence for Recent Admixture,” <i><b>The American Journal of Human Genetics</b></i>, vol. 98, no. 2, pp. 396–397, Feb. 2016, doi: 10.1016/j.ajhg.2015.12.025.</div>
									</div>
									<div data-csl-entry-id="montinaro_complex_2016" class="csl-entry">
									  <div class="csl-left-margin">[16]</div><div class="csl-right-inline">F. Montinaro <i>et al.</i>, “Complex Ancient Genetic Structure and Cultural Transitions in Southern African Populations,” <i><b>Genetics</b></i>, vol. 205, no. 1, pp. 303–316, 2017, doi: 10.1534/genetics.116.189209.</div>
									</div>
									<div data-csl-entry-id="busby_admixture_2016" class="csl-entry">
									  <div class="csl-left-margin">[17]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b> <i>et al.</i>, “Admixture into and within sub-Saharan Africa,” <i><b>eLife</b></i>, vol. 5, p. e15266, Jun. 2016, doi: 10.7554/eLife.15266.</div>
									</div>
									<div data-csl-entry-id="leffler_resistance_2017" class="csl-entry">
									  <div class="csl-left-margin">[18]</div><div class="csl-right-inline">E. M. Leffler <i>et al.</i>, “Resistance to malaria through structural variation of red blood cell invasion receptors,” <i><b>Science</b></i>, p. eaam6393, May 2017, doi: 10.1126/science.aam6393.</div>
									</div>
									<div data-csl-entry-id="NDILA2018e333" class="csl-entry">
									  <div class="csl-left-margin">[19]</div><div class="csl-right-inline">C. M. Ndila <i>et al.</i>, “Human candidate gene polymorphisms and risk of severe malaria in children in Kilifi, Kenya: a case-control association study,” <i><b>The Lancet Haematology</b></i>, vol. 5, no. 8, pp. e333–e345, 2018, doi: https://doi.org/10.1016/S2352-3026(18)30107-8.</div>
									</div>
									<div data-csl-entry-id="bandInsightsMalariaSusceptibility2019" class="csl-entry">
									  <div class="csl-left-margin">[20]</div><div class="csl-right-inline">G. Band <i>et al.</i>, “Insights into malaria susceptibility using genome-wide data on 17,000 individuals from Africa, Asia and Oceania,” <i><b>Nature Communications</b></i>, vol. 10, no. 1, p. 5732, Dec. 2019, doi: 10.1038/s41467-019-13480-z.</div>
									</div>
									<div data-csl-entry-id="bollialessandroRiskCoronaryArtery2021" class="csl-entry">
									  <div class="csl-left-margin">[21]</div><div class="csl-right-inline">A. Bolli, P. Di Domenico, R. Pastorino, <b>G.</b> <b>Busby</b>, and G. Bottà, “Risk of Coronary Artery Disease Conferred by Low-Density Lipoprotein Cholesterol Depends on Polygenic Background,” <i><b>Circulation</b></i>, vol. 143, no., 2021, doi: 10.1161/CIRCULATIONAHA.120.051843.</div>
									</div>
									<div data-csl-entry-id="mujwaraIntegratingPolygenicRisk2022" class="csl-entry">
									  <div class="csl-left-margin">[22]</div><div class="csl-right-inline">D. Mujwara <i>et al.</i>, “Integrating a Polygenic Risk Score for Coronary Artery Disease as a Risk-Enhancing Factor in the Pooled Cohort Equation: A Cost-Effectiveness Analysis Study,” <i><b>Journal of the American Heart Association</b></i>, vol., no., p. e025236, 2022, doi: 10.1161/JAHA.121.025236.</div>
									</div>
									<div data-csl-entry-id="mwendaDetection351SARSCoV22021" class="csl-entry">
									  <div class="csl-left-margin">[23]</div><div class="csl-right-inline">M. Mwenda, “Detection of B.1.351 SARS-CoV-2 Variant Strain — Zambia, December 2020,” <i><b>MMWR. Morbidity and Mortality Weekly Report</b></i>, vol. 70, 2021, doi: 10.15585/mmwr.mm7008e2.</div>
									</div>
									<div data-csl-entry-id="elliottFineScaleGeneticStructure2022" class="csl-entry">
									  <div class="csl-left-margin">[24]</div><div class="csl-right-inline">K. S. Elliott <i>et al.</i>, “Fine-Scale Genetic Structure in the United Arab Emirates Reflects Endogamous and Consanguineous Culture, Population History, and Geography,” <i><b>Molecular Biology and Evolution</b></i>, vol. 39, no. 3, p. msac039, Mar. 2022, doi: 10.1093/molbev/msac039.</div>
									</div>
									<div data-csl-entry-id="mujwara_2023" class="csl-entry">
									  <div class="csl-left-margin">[25]</div><div class="csl-right-inline">D. Mujwara, J. Kintzle, P. Di Domenico, <b>G. B.</b> <b>Busby</b>, and G. Bottà, “Cost-effectiveness analysis of implementing polygenic risk score in a workplace cardiovascular disease prevention program,” <i><b>Frontiers in Public Health</b></i>, vol. 11, 2023, doi: 10.3389/fpubh.2023.1139496.</div>
									</div>
									<div data-csl-entry-id="busbyAncestryspecificPolygenicRisk2023" class="csl-entry">
									  <div class="csl-left-margin">[26]</div><div class="csl-right-inline"><b>G. B.</b> <b>Busby</b>, S. Kulm, A. Bolli, J. Kintzle, P. D. Domenico, and G. Bottà, “Ancestry-Specific Polygenic Risk Scores Are Risk Enhancers for Clinical Cardiovascular Disease Assessments,” <i><b>Nature Communications</b></i>, vol. 14, no. 1, p. 7105, Nov. 2023, doi: 10.1038/s41467-023-42897-w.</div>
									</div>
									<div data-csl-entry-id="decesareFlexibleCosteffectiveGenomic2024" class="csl-entry">
									  <div class="csl-left-margin">[27]</div><div class="csl-right-inline">M. de Cesare <i>et al.</i>, “Flexible and Cost-Effective Genomic Surveillance of P. Falciparum Malaria with Targeted Nanopore Sequencing,” <i>Nature Communications</i>, vol. 15, no. 1, p. 1413, Feb. 2024, doi: 10.1038/s41467-024-45688-z.</div>
									</div>
									<div data-csl-entry-id="busbyGeneticAssessmentsBreast2021" class="csl-entry">
									  <div class="csl-left-margin">[28]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b>, P. Craig, N. Yousfi, S. Hebbalkar, P. Di Domenico, and G. Bottà, “Genetic Assessments of Breast Cancer Risk That Do Not Account for Polygenic Background Are Incomplete and Lead to Incorrect Preventative Strategies,” <i>medRxiv</i>, p. 2021.08.13.21262050, 2021, doi: 10.1101/2021.08.13.21262050.</div>
									</div>
									<div data-csl-entry-id="busby_inferring_2017" class="csl-entry">
									  <div class="csl-left-margin">[29]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b> <i>et al.</i>, “Inferring adaptive gene-flow in recent African history,” <i><b>bioRxiv</b></i>, Jan. 2017, doi: 10.1101/205252.</div>
									</div>
									<div data-csl-entry-id="10.1093/clinchem/hvad049" class="csl-entry">
									  <div class="csl-left-margin">[30]</div><div class="csl-right-inline">T. J. Hayeck, <b>G. B. J.</b> <b>Busby</b>, S. Chun, A. C. F. Lewis, M. C. Roberts, and B. J. Vilhjálmsson, “Polygenic Risk Scores: Genomes to Risk Prediction,” <i><b>Clinical Chemistry</b></i>, vol. 69, no. 6, pp. 551–557, May 2023, doi: 10.1093/clinchem/hvad049.</div>
									</div>
									<div data-csl-entry-id="haffenerAdaptiveAdmixtureACKR12024" class="csl-entry">
									  <div class="csl-left-margin">[31]</div><div class="csl-right-inline">P. E. Haffener <i>et al.</i>, <i>Adaptive Admixture at ACKR1 (the Duffy Locus) May Have Shaped Plasmodium Vivax Prevalence in Oman</i>. bioRxiv, 2024, p. 2024.03.06.583766. doi: 10.1101/2024.03.06.583766.</div>
									</div>
									<div data-csl-entry-id="busbyAncestryspecificPolygenicRisk2024" class="csl-entry">
									  <div class="csl-left-margin">[32]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b> <i>et al.</i>, <i>Ancestry-Specific Polygenic Risk Scores Improve Clinical Assessments of Breast Cancer Risk in Diverse Populations</i>. Research Square, 2024. doi: 10.21203/rs.3.rs-4022359/v1.</div>
									</div>
									<div data-csl-entry-id="diagnostics14161826" class="csl-entry">
									  <div class="csl-left-margin">[33]</div><div class="csl-right-inline">N. Tsoulos <i>et al.</i>, “Polygenic Risk Score (PRS) Combined with NGS Panel Testing Increases Accuracy in Hereditary Breast Cancer Risk Estimation,” <i>Diagnostics</i>, vol. 14, no. 16, 2024, doi: 10.3390/diagnostics14161826.</div>
									</div>
								  </div>

								</p>

							</article>

						<!-- About -->
							<article id="about">
								<h2 class="major">CV</h2>
								<span class="image main"><img src="images/pic03.jpg" alt="" /></span>
								<h3 class="major">Employment</h3>
									<div class="table-wrapper">
										<table>
											<thead>
												<tr>
													<th>Date</th>
													<th>Role</th>
													<th>Employer</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Feb 2021 - present</td>
													<td>Chief Scientific Officer and Co‐founder</td>
													<td>Allelica Inc</td>
												</tr>
												<tr>
													<td>Oct 2020 - Jan 2021</td>
													<td>Principal Research Scientist, NOMADS</td>
													<td>CHG, Oxford University</td>
												</tr>
												<tr>
													<td>May 2017 - Sep 2020</td>
													<td>Senior Research Associate in Translational Genomics</td>
													<td>BDI, Oxford University</td>
												</tr>
												<tr>
													<td>Jul 2013 - Apr 2017r</td>
													<td>Research Associate in Statistical Genomics</td>
													<td>CHG, Oxford University</td>
												</tr>
												<tr>
													<td>Oct 2014 - Sep 2014</td>
													<td>Non‐Stipendiary Lectureship in Human Sciences</td>
													<td>St Hugh's College, Oxford University</td>
												</tr>
												<tr>
													<td>Sep 2008 - Jun 2013</td>
													<td>Graduate Research Assistant, Tutor and Demonstrator</td>
													<td>Dept Zoology, Oxford University</td>
												</tr>
												<tr>
													<td>Sep 2007 - Sep 2008</td>
													<td>Conservation Genetics Research Technician</td>
													<td>Insitute of Zoology, London Zoo</td>
												</tr>
												<tr>
													<td>Sep 2006 - Sep 2007</td>
													<td>Risk Analyst</td>
													<td>Risk Management Solutions, London</td>
												</tr>
											</tbody>
											
										</table>
									</div>

									<h3 class="major">Education</h3>
									<div class="table-wrapper">
										<table>
											<thead>
												<tr>
													<th>Date</th>
													<th>Subject</th>
													<th>Insitute</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Oct 2008 - Jun 2012</td>
													<td>DPhil Human Evolutionary Genetics</td>
													<td>Oxford University</td>
												</tr>
												<tr>
													<td>Oct 2005 - Sep 2006</td>
													<td>MRes Ecology Evolution and Conservation (Distinction)</td>
													<td>Imperial College London</td>
												</tr>
												<tr>
													<td>Oct 2001 - Jul 2005</td>
													<td>B.Sc (Hons) Zoology (1st Class)</td>
													<td>University of Edinburgh</td>
												</tr>
	
											</tbody>
											
										</table>
									</div>

									<section>
										<h3 class="major">Awards</h3>
										<ul class="alt">
											<li>2013 | Fellowship of the Royal Geographical Society</li>
											<li>2008 | Somerville College Scholarship</li>
											<li>2007 | Postgraduate Fellowship of the Royal Geographical Society</li>
										</ul>
									</section>
								
							</article>

						<!-- Contact -->
							<article id="contact">
								<h2 class="major">Contact</h2>
								<form method="post" action="#">
									<div class="fields">
										<div class="field half">
											<label for="name">Name</label>
											<input type="text" name="name" id="name" />
										</div>
										<div class="field half">
											<label for="email">Email</label>
											<input type="text" name="email" id="email" />
										</div>
										<div class="field">
											<label for="message">Message</label>
											<textarea name="message" id="message" rows="4"></textarea>
										</div>
									</div>
									<ul class="actions">
										<li><input type="submit" value="Send Message" class="primary" /></li>
										<li><input type="reset" value="Reset" /></li>
									</ul>
								</form>
								<ul class="icons">
									<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
									<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
									<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
									<li><a href="#" class="icon brands fa-github"><span class="label">GitHub</span></a></li>
								</ul>
							</article>

						<!-- Elements -->
							<article id="elements">
								<h2 class="major">Elements</h2>

								<section>
									<h3 class="major">Text</h3>
									<p>This is <b>bold</b> and this is <strong>strong</strong>. This is <i>italic</i> and this is <em>emphasized</em>.
									This is <sup>superscript</sup> text and this is <sub>subscript</sub> text.
									This is <u>underlined</u> and this is code: <code>for (;;) { ... }</code>. Finally, <a href="#">this is a link</a>.</p>
									<hr />
									<h2>Heading Level 2</h2>
									<h3>Heading Level 3</h3>
									<h4>Heading Level 4</h4>
									<h5>Heading Level 5</h5>
									<h6>Heading Level 6</h6>
									<hr />
									<h4>Blockquote</h4>
									<blockquote>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan faucibus. Vestibulum ante ipsum primis in faucibus lorem ipsum dolor sit amet nullam adipiscing eu felis.</blockquote>
									<h4>Preformatted</h4>
									<pre><code>i = 0;

while (!deck.isInOrder()) {
    print 'Iteration ' + i;
    deck.shuffle();
    i++;
}

print 'It took ' + i + ' iterations to sort the deck.';</code></pre>
								</section>

								<section>
									<h3 class="major">Lists</h3>

									<h4>Unordered</h4>
									<ul>
										<li>Dolor pulvinar etiam.</li>
										<li>Sagittis adipiscing.</li>
										<li>Felis enim feugiat.</li>
									</ul>

									<h4>Alternate</h4>
									<ul class="alt">
										<li>Dolor pulvinar etiam.</li>
										<li>Sagittis adipiscing.</li>
										<li>Felis enim feugiat.</li>
									</ul>

									<h4>Ordered</h4>
									<ol>
										<li>Dolor pulvinar etiam.</li>
										<li>Etiam vel felis viverra.</li>
										<li>Felis enim feugiat.</li>
										<li>Dolor pulvinar etiam.</li>
										<li>Etiam vel felis lorem.</li>
										<li>Felis enim et feugiat.</li>
									</ol>
									<h4>Icons</h4>
									<ul class="icons">
										<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
										<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
										<li><a href="#" class="icon brands fa-github"><span class="label">Github</span></a></li>
									</ul>

									<h4>Actions</h4>
									<ul class="actions">
										<li><a href="#" class="button primary">Default</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
									<ul class="actions stacked">
										<li><a href="#" class="button primary">Default</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
								</section>

								<section>
									<h3 class="major">Table</h3>
									<h4>Default</h4>
									<div class="table-wrapper">
										<table>
											<thead>
												<tr>
													<th>Name</th>
													<th>Description</th>
													<th>Price</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Item One</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Two</td>
													<td>Vis ac commodo adipiscing arcu aliquet.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Three</td>
													<td> Morbi faucibus arcu accumsan lorem.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Four</td>
													<td>Vitae integer tempus condimentum.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Five</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
											</tbody>
											<tfoot>
												<tr>
													<td colspan="2"></td>
													<td>100.00</td>
												</tr>
											</tfoot>
										</table>
									</div>

									<h4>Alternate</h4>
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Name</th>
													<th>Description</th>
													<th>Price</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Item One</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Two</td>
													<td>Vis ac commodo adipiscing arcu aliquet.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Three</td>
													<td> Morbi faucibus arcu accumsan lorem.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Four</td>
													<td>Vitae integer tempus condimentum.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Five</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
											</tbody>
											<tfoot>
												<tr>
													<td colspan="2"></td>
													<td>100.00</td>
												</tr>
											</tfoot>
										</table>
									</div>
								</section>

								<section>
									<h3 class="major">Buttons</h3>
									<ul class="actions">
										<li><a href="#" class="button primary">Primary</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
									<ul class="actions">
										<li><a href="#" class="button">Default</a></li>
										<li><a href="#" class="button small">Small</a></li>
									</ul>
									<ul class="actions">
										<li><a href="#" class="button primary icon solid fa-download">Icon</a></li>
										<li><a href="#" class="button icon solid fa-download">Icon</a></li>
									</ul>
									<ul class="actions">
										<li><span class="button primary disabled">Disabled</span></li>
										<li><span class="button disabled">Disabled</span></li>
									</ul>
								</section>

								<section>
									<h3 class="major">Form</h3>
									<form method="post" action="#">
										<div class="fields">
											<div class="field half">
												<label for="demo-name">Name</label>
												<input type="text" name="demo-name" id="demo-name" value="" placeholder="Jane Doe" />
											</div>
											<div class="field half">
												<label for="demo-email">Email</label>
												<input type="email" name="demo-email" id="demo-email" value="" placeholder="jane@untitled.tld" />
											</div>
											<div class="field">
												<label for="demo-category">Category</label>
												<select name="demo-category" id="demo-category">
													<option value="">-</option>
													<option value="1">Manufacturing</option>
													<option value="1">Shipping</option>
													<option value="1">Administration</option>
													<option value="1">Human Resources</option>
												</select>
											</div>
											<div class="field half">
												<input type="radio" id="demo-priority-low" name="demo-priority" checked>
												<label for="demo-priority-low">Low</label>
											</div>
											<div class="field half">
												<input type="radio" id="demo-priority-high" name="demo-priority">
												<label for="demo-priority-high">High</label>
											</div>
											<div class="field half">
												<input type="checkbox" id="demo-copy" name="demo-copy">
												<label for="demo-copy">Email me a copy</label>
											</div>
											<div class="field half">
												<input type="checkbox" id="demo-human" name="demo-human" checked>
												<label for="demo-human">Not a robot</label>
											</div>
											<div class="field">
												<label for="demo-message">Message</label>
												<textarea name="demo-message" id="demo-message" placeholder="Enter your message" rows="6"></textarea>
											</div>
										</div>
										<ul class="actions">
											<li><input type="submit" value="Send Message" class="primary" /></li>
											<li><input type="reset" value="Reset" /></li>
										</ul>
									</form>
								</section>

							</article>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<p class="copyright">&copy; George Busby 2024: <a href="https://html5up.net">HTML5 UP</a>.</p>
					</footer>

			</div>

		<!-- BG -->
			<div id="bg"></div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
