<div class="csl-bib-body">
  <div data-csl-entry-id="busby_genetic_2009" class="csl-entry">
    <div class="csl-left-margin">[1]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b> <i>et al.</i>, “Genetic analysis of scat reveals leopard Panthera pardus and cheetah Acinonyx jubatus in southern Algeria,” <i><b>Oryx</b></i>, vol. 43, no. 3, pp. 412–415, 2009.</div>
  </div>
  <div data-csl-entry-id="busby_peopling_2012" class="csl-entry">
    <div class="csl-left-margin">[2]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b> <i>et al.</i>, “The peopling of Europe and the cautionary tale of Y chromosome lineage R-M269,” <i><b>Proceedings of the Royal Society B: Biological Sciences</b></i>, vol. 279, no. 1730, pp. 884–892, Mar. 2012, doi: 10.1098/rspb.2011.1044.</div>
  </div>
  <div data-csl-entry-id="busby_finding_2006" class="csl-entry">
    <div class="csl-left-margin">[3]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b>, “Finding the Blues: an investigation into the origin and evolution of African American music,” MRes, Imperial College, London,UK, 2006.</div>
  </div>
  <div data-csl-entry-id="bertoncini_y_2012" class="csl-entry">
    <div class="csl-left-margin">[4]</div><div class="csl-right-inline">S. Bertoncini <i>et al.</i>, “A Y variant which traces the genetic heritage of ligures tribes,” <i><b>Journal of Biological Research - Bollettino della Società Italiana di Biologia Sperimentale</b></i>, vol. 85, no. 1, Nov. 2012, doi: 10.4081/jbr.2012.4087.</div>
  </div>
  <div data-csl-entry-id="coia_demographic_2013" class="csl-entry">
    <div class="csl-left-margin">[5]</div><div class="csl-right-inline">V. Coia <i>et al.</i>, “Demographic Histories, Isolation and Social Factors as Determinants of the Genetic Structure of Alpine Linguistic Groups,” <i><b>PLoS ONE</b></i>, vol. 8, no. 12, p. e81704, Dec. 2013, doi: 10.1371/journal.pone.0081704.</div>
  </div>
  <div data-csl-entry-id="hellenthal_genetic_2014" class="csl-entry">
    <div class="csl-left-margin">[6]</div><div class="csl-right-inline">G. Hellenthal <i>et al.</i>, “A Genetic Atlas of Human Admixture History,” <i><b>Science</b></i>, vol. 343, no. 6172, pp. 747–751, Feb. 2014, doi: 10.1126/science.1243518.</div>
  </div>
  <div data-csl-entry-id="marks_static_2014" class="csl-entry">
    <div class="csl-left-margin">[7]</div><div class="csl-right-inline">S. J. Marks <i>et al.</i>, “Static and moving frontiers: the genetic landscape of Southern African Bantu-speaking populations,” <i><b>Molecular Biology and Evolution</b></i>, p. msu263, Sep. 2014, doi: 10.1093/molbev/msu263.</div>
  </div>
  <div data-csl-entry-id="lazaridis_ancient_2014" class="csl-entry">
    <div class="csl-left-margin">[8]</div><div class="csl-right-inline">I. Lazaridis <i>et al.</i>, “Ancient human genomes suggest three ancestral populations for present-day Europeans,” <i><b>Nature</b></i>, vol. 513, no. 7518, pp. 409–413, Sep. 2014, doi: 10.1038/nature13673.</div>
  </div>
  <div data-csl-entry-id="cini_social_2015" class="csl-entry">
    <div class="csl-left-margin">[9]</div><div class="csl-right-inline">A. Cini, S. Patalano, A. Segonds-Pichon, <b>G.</b> <b>Busby</b>, R. Cervo, and S. Sumner, “Social parasitism and the molecular basis of phenotypic evolution,” <i><b>Frontiers in Genetics</b></i>, vol. 6, p. 32, 2015, doi: 10.3389/fgene.2015.00032.</div>
  </div>
  <div data-csl-entry-id="montinaro_unravelling_2015" class="csl-entry">
    <div class="csl-left-margin">[10]</div><div class="csl-right-inline">F. Montinaro, <b>G.</b> <b>Busby</b>, V. L. Pascali, S. Myers, G. Hellenthal, and C. Capelli, “Unravelling the hidden ancestry of American admixed populations,” <i><b>Nature Communications</b></i>, vol. 6, Mar. 2015, doi: 10.1038/ncomms7596.</div>
  </div>
  <div data-csl-entry-id="tofanelli_greeks_2015" class="csl-entry">
    <div class="csl-left-margin">[11]</div><div class="csl-right-inline">S. Tofanelli <i>et al.</i>, “The Greeks in the West: genetic signatures of the Hellenic colonisation in southern Italy and Sicily,” <i><b>European Journal of Human Genetics</b></i>, Jul. 2015, doi: 10.1038/ejhg.2015.124.</div>
  </div>
  <div data-csl-entry-id="martinez-cadenas_relationship_2015" class="csl-entry">
    <div class="csl-left-margin">[12]</div><div class="csl-right-inline">C. Martinez-Cadenas <i>et al.</i>, “The relationship between surname frequency and Y chromosome variation in Spain,” <i><b>European Journal of Human Genetics</b></i>, Apr. 2015, doi: 10.1038/ejhg.2015.75.</div>
  </div>
  <div data-csl-entry-id="busby_role_2015" class="csl-entry">
    <div class="csl-left-margin">[13]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b> <i>et al.</i>, “The Role of Recent Admixture in Forming the Contemporary West Eurasian Genomic Landscape,” <i><b>Current Biology</b></i>, vol. 25, no. 19, pp. 2518–2526, May 2015, doi: 10.1016/j.cub.2015.08.007.</div>
  </div>
  <div data-csl-entry-id="gonzalez-santos_genome-wide_2015" class="csl-entry">
    <div class="csl-left-margin">[14]</div><div class="csl-right-inline">M. González-Santos <i>et al.</i>, “Genome-Wide SNP Analysis of Southern African Populations Provides New Insights into the Dispersal of Bantu-Speaking Groups,” <i><b>Genome Biology and Evolution</b></i>, vol. 7, no. 9, pp. 2560–2568, Sep. 2015, doi: 10.1093/gbe/evv164.</div>
  </div>
  <div data-csl-entry-id="hellenthal_kalash_2016" class="csl-entry">
    <div class="csl-left-margin">[15]</div><div class="csl-right-inline">G. Hellenthal <i>et al.</i>, “The Kalash Genetic Isolate? The Evidence for Recent Admixture,” <i><b>The American Journal of Human Genetics</b></i>, vol. 98, no. 2, pp. 396–397, Feb. 2016, doi: 10.1016/j.ajhg.2015.12.025.</div>
  </div>
  <div data-csl-entry-id="montinaro_complex_2016" class="csl-entry">
    <div class="csl-left-margin">[16]</div><div class="csl-right-inline">F. Montinaro <i>et al.</i>, “Complex Ancient Genetic Structure and Cultural Transitions in Southern African Populations,” <i><b>Genetics</b></i>, vol. 205, no. 1, pp. 303–316, 2017, doi: 10.1534/genetics.116.189209.</div>
  </div>
  <div data-csl-entry-id="busby_admixture_2016" class="csl-entry">
    <div class="csl-left-margin">[17]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b> <i>et al.</i>, “Admixture into and within sub-Saharan Africa,” <i><b>eLife</b></i>, vol. 5, p. e15266, Jun. 2016, doi: 10.7554/eLife.15266.</div>
  </div>
  <div data-csl-entry-id="leffler_resistance_2017" class="csl-entry">
    <div class="csl-left-margin">[18]</div><div class="csl-right-inline">E. M. Leffler <i>et al.</i>, “Resistance to malaria through structural variation of red blood cell invasion receptors,” <i><b>Science</b></i>, p. eaam6393, May 2017, doi: 10.1126/science.aam6393.</div>
  </div>
  <div data-csl-entry-id="NDILA2018e333" class="csl-entry">
    <div class="csl-left-margin">[19]</div><div class="csl-right-inline">C. M. Ndila <i>et al.</i>, “Human candidate gene polymorphisms and risk of severe malaria in children in Kilifi, Kenya: a case-control association study,” <i><b>The Lancet Haematology</b></i>, vol. 5, no. 8, pp. e333–e345, 2018, doi: https://doi.org/10.1016/S2352-3026(18)30107-8.</div>
  </div>
  <div data-csl-entry-id="bandInsightsMalariaSusceptibility2019" class="csl-entry">
    <div class="csl-left-margin">[20]</div><div class="csl-right-inline">G. Band <i>et al.</i>, “Insights into malaria susceptibility using genome-wide data on 17,000 individuals from Africa, Asia and Oceania,” <i><b>Nature Communications</b></i>, vol. 10, no. 1, p. 5732, Dec. 2019, doi: 10.1038/s41467-019-13480-z.</div>
  </div>
  <div data-csl-entry-id="bollialessandroRiskCoronaryArtery2021" class="csl-entry">
    <div class="csl-left-margin">[21]</div><div class="csl-right-inline">A. Bolli, P. Di Domenico, R. Pastorino, <b>G.</b> <b>Busby</b>, and G. Bottà, “Risk of Coronary Artery Disease Conferred by Low-Density Lipoprotein Cholesterol Depends on Polygenic Background,” <i><b>Circulation</b></i>, vol. 143, no., 2021, doi: 10.1161/CIRCULATIONAHA.120.051843.</div>
  </div>
  <div data-csl-entry-id="mujwaraIntegratingPolygenicRisk2022" class="csl-entry">
    <div class="csl-left-margin">[22]</div><div class="csl-right-inline">D. Mujwara <i>et al.</i>, “Integrating a Polygenic Risk Score for Coronary Artery Disease as a Risk-Enhancing Factor in the Pooled Cohort Equation: A Cost-Effectiveness Analysis Study,” <i><b>Journal of the American Heart Association</b></i>, vol., no., p. e025236, 2022, doi: 10.1161/JAHA.121.025236.</div>
  </div>
  <div data-csl-entry-id="mwendaDetection351SARSCoV22021" class="csl-entry">
    <div class="csl-left-margin">[23]</div><div class="csl-right-inline">M. Mwenda, “Detection of B.1.351 SARS-CoV-2 Variant Strain — Zambia, December 2020,” <i><b>MMWR. Morbidity and Mortality Weekly Report</b></i>, vol. 70, 2021, doi: 10.15585/mmwr.mm7008e2.</div>
  </div>
  <div data-csl-entry-id="elliottFineScaleGeneticStructure2022" class="csl-entry">
    <div class="csl-left-margin">[24]</div><div class="csl-right-inline">K. S. Elliott <i>et al.</i>, “Fine-Scale Genetic Structure in the United Arab Emirates Reflects Endogamous and Consanguineous Culture, Population History, and Geography,” <i><b>Molecular Biology and Evolution</b></i>, vol. 39, no. 3, p. msac039, Mar. 2022, doi: 10.1093/molbev/msac039.</div>
  </div>
  <div data-csl-entry-id="mujwara_2023" class="csl-entry">
    <div class="csl-left-margin">[25]</div><div class="csl-right-inline">D. Mujwara, J. Kintzle, P. Di Domenico, <b>G. B.</b> <b>Busby</b>, and G. Bottà, “Cost-effectiveness analysis of implementing polygenic risk score in a workplace cardiovascular disease prevention program,” <i><b>Frontiers in Public Health</b></i>, vol. 11, 2023, doi: 10.3389/fpubh.2023.1139496.</div>
  </div>
  <div data-csl-entry-id="busbyAncestryspecificPolygenicRisk2023" class="csl-entry">
    <div class="csl-left-margin">[26]</div><div class="csl-right-inline"><b>G. B.</b> <b>Busby</b>, S. Kulm, A. Bolli, J. Kintzle, P. D. Domenico, and G. Bottà, “Ancestry-Specific Polygenic Risk Scores Are Risk Enhancers for Clinical Cardiovascular Disease Assessments,” <i><b>Nature Communications</b></i>, vol. 14, no. 1, p. 7105, Nov. 2023, doi: 10.1038/s41467-023-42897-w.</div>
  </div>
  <div data-csl-entry-id="decesareFlexibleCosteffectiveGenomic2024" class="csl-entry">
    <div class="csl-left-margin">[27]</div><div class="csl-right-inline">M. de Cesare <i>et al.</i>, “Flexible and Cost-Effective Genomic Surveillance of P. Falciparum Malaria with Targeted Nanopore Sequencing,” <i>Nature Communications</i>, vol. 15, no. 1, p. 1413, Feb. 2024, doi: 10.1038/s41467-024-45688-z.</div>
  </div>
  <div data-csl-entry-id="busbyGeneticAssessmentsBreast2021" class="csl-entry">
    <div class="csl-left-margin">[28]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b>, P. Craig, N. Yousfi, S. Hebbalkar, P. Di Domenico, and G. Bottà, “Genetic Assessments of Breast Cancer Risk That Do Not Account for Polygenic Background Are Incomplete and Lead to Incorrect Preventative Strategies,” <i>medRxiv</i>, p. 2021.08.13.21262050, 2021, doi: 10.1101/2021.08.13.21262050.</div>
  </div>
  <div data-csl-entry-id="busby_inferring_2017" class="csl-entry">
    <div class="csl-left-margin">[29]</div><div class="csl-right-inline"><b>G.</b> <b>Busby</b> <i>et al.</i>, “Inferring adaptive gene-flow in recent African history,” <i><b>bioRxiv</b></i>, Jan. 2017, doi: 10.1101/205252.</div>
  </div>
  <div data-csl-entry-id="10.1093/clinchem/hvad049" class="csl-entry">
    <div class="csl-left-margin">[30]</div><div class="csl-right-inline">T. J. Hayeck, <b>G. B. J.</b> <b>Busby</b>, S. Chun, A. C. F. Lewis, M. C. Roberts, and B. J. Vilhjálmsson, “Polygenic Risk Scores: Genomes to Risk Prediction,” <i><b>Clinical Chemistry</b></i>, vol. 69, no. 6, pp. 551–557, May 2023, doi: 10.1093/clinchem/hvad049.</div>
  </div>
  <div data-csl-entry-id="haffenerAdaptiveAdmixtureACKR12024" class="csl-entry">
    <div class="csl-left-margin">[31]</div><div class="csl-right-inline">P. E. Haffener <i>et al.</i>, <i>Adaptive Admixture at ACKR1 (the Duffy Locus) May Have Shaped Plasmodium Vivax Prevalence in Oman</i>. bioRxiv, 2024, p. 2024.03.06.583766. doi: 10.1101/2024.03.06.583766.</div>
  </div>
  <div data-csl-entry-id="busbyAncestryspecificPolygenicRisk2024" class="csl-entry">
    <div class="csl-left-margin">[32]</div><div class="csl-right-inline"><b>G. B. J.</b> <b>Busby</b> <i>et al.</i>, <i>Ancestry-Specific Polygenic Risk Scores Improve Clinical Assessments of Breast Cancer Risk in Diverse Populations</i>. Research Square, 2024. doi: 10.21203/rs.3.rs-4022359/v1.</div>
  </div>
  <div data-csl-entry-id="diagnostics14161826" class="csl-entry">
    <div class="csl-left-margin">[33]</div><div class="csl-right-inline">N. Tsoulos <i>et al.</i>, “Polygenic Risk Score (PRS) Combined with NGS Panel Testing Increases Accuracy in Hereditary Breast Cancer Risk Estimation,” <i>Diagnostics</i>, vol. 14, no. 16, 2024, doi: 10.3390/diagnostics14161826.</div>
  </div>
</div>